SmartPlaces.Facilities/lib/IngestionManager/src/IngestionProcessorBase.cs

```diff
@@ -265,6 +265,10 @@ protected bool TryGetOutputInterfaceDtmi(Dtmi inputDtmi, out Dtmi? outputDtmi)
                     // Populate the content of the twin
                     var contentDictionary = new Dictionary<string, object>();
 
+                    // Add the alternate classification to the twin (brickSchema)
+                    var alternateClassification = new { BrickSchema = inputDtmi.AbsoluteUri };
```

Comment:
Neither the source nor the target ontologies are required to be Brick. Maybe `SourceSchema`,  `AlternateSchema` or `OriginalSchema`?

Reply:
Unfortunately, right now, the property under the alternateClassification entity in the Willow DTO is called "BrickSchema". 

[Monday 5:24 PM] Rick Szcodronski

We have a property called alternateClassification which is meant to be where we toss other schemas. It already has brickSchema.

 This is one of those cases that demands another level of redirection be implemented... but if we have to do that, we'll probably fork the repo to make a Willow-specific ingestion manager.

    {

      "@type": "Property",

      "name": "alternateClassification",

      "displayName": {

        "en": "Alternate Classification"

      },

      "writable": true,

      "schema": {

        "@type": "Object",

        "fields": [

          {

            "name": "masterFormat",

            "displayName": {

              "en": "MasterFormat"

            },

            "schema": "dtmi:com:willowinc:SpaceAlternateClassificationObject;1"

          },

          {

            "name": "omniClass",

            "displayName": {

              "en": "OmniClass"

            },

            "schema": "dtmi:com:willowinc:SpaceAlternateClassificationObject;1"

          },

          {

            "name": "uniFormat",

            "displayName": {

              "en": "UniFormat"

            },

            "schema": "dtmi:com:willowinc:SpaceAlternateClassificationObject;1"

          },

          {

            "name": "uniclass",

            "displayName": {

              "en": "Uniclass"

            },

            "schema": "dtmi:com:willowinc:SpaceAlternateClassificationObject;1"

          },

          {

            "name": "projectHaystack",

            "displayName": {

              "en": "Project Haystack"

            },

            "schema": "dtmi:com:willowinc:SpaceAlternateClassificationObject;1"

          },

          {

            "name": "brickSchema",

            "displayName": {

              "en": "Brick Schema"

            },

            "schema": "dtmi:com:willowinc:SpaceAlternateClassificationObject;1"

          }

        ]

      }

Reply:
Came up with an alternative approach to allow Willow to inject custom properties as needed.